pages/moderator/[jam].js

```diff
@@ -144,6 +146,17 @@ const Jam = () => {
     setNewStatements(
       jam.statements.filter((statement) => statement.state === 0),
     );
+
+    setTotalVotes(
+      jam.statements.reduce((acc, statement) => {
+        return (
+          acc +
+          (statement.numAgrees || 0) +
+          (statement.numDisagrees || 0) +
+          (statement.numSkipped || 0)
```

Comment:
should skips be included in the total votes?

Reply:
Good point - I thought of skipping as an action but you're right, I'll remove.